code/modules/virus2/disease2.dm

```diff
@@ -88,6 +88,13 @@
 		else
 			resistance += rand(1,9)
 
+	//VOREStation Add Start - Corophazine can treat higher stages
```

Comment:
This feels out of scope. Why?

Reply:
This was done because Aronai requested that the virus problem be dealt with by some other way than the Field Medic going into the Virology lab.

See: https://github.com/VOREStation/VOREStation/pull/4451#issuecomment-432458597